src/pages/SendCredentials/SendCredentials.js

```diff
@@ -70,7 +69,7 @@ const Verifiers = () => {
 	useEffect(() => {
 		const filtered = verifiers.filter((verifier) => {
 			const name = verifier.name.toLowerCase();
-			const query = searchQuery.toLowerCase();
+			const query = searchQuery.toLowerCase().trimStart();
```

Comment:
same as `AddCredential.js` for here and L151

Reply:
We're abstracting the search input to a `SearchComponent` and we'll check that. I believe it is still necessary, because of the highlighting expression. If we start with empty space (e.g. by accident), it won't highlight the following existing characters.

Reply:
both ideas make perfect sense and yes, at this point the search/filter criteria will remain the same across all components so feel free re-organize them a bit